lib/api/abis/bridge/L1BondingManager.ts

```diff
@@ -97,6 +97,21 @@ export const l1BondingManager = {
       stateMutability: "view",
       type: "function",
     },
+    {
```

Comment:
For consistency reasons, I added this function to both the `bridge/L1BondingManger` and the `main/BondingManger`.